src/components/popup/PopupDirective.js

```diff
@@ -71,18 +76,15 @@ goog.require('ga_window_service');
           scope.options.isReduced = false;
 
           // Per default hide the print function
-          if (!angular.isDefined(scope.options.showPrint) ||
-              gaBrowserSniffer.mobile) {
+          if (!angular.isDefined(scope.options.showPrint)) {
```

Comment:
Not very useful but it works also on mobile. I don't think a user might be offended by this button. 

I can also hide it on small screen, if we receive complain about it.